build.xml

```diff
@@ -6,7 +6,7 @@
         build.xml file for JGroups. Needs Ant (jakarta.apache.org) to run
     </description>
 
-    <property name="version" value="2.12.4.Final-TEK"/>
+    <property name="version" value="2.12.4.Final-TEK-1464-1420"/>
```

Comment:
Ok that's fine. I would suggest add a comment to this pom file regarding the 2 fixes then.